src/NUnitConsole/nunit3-console.tests/ResultReporterTests.cs

```diff
@@ -46,12 +45,21 @@ public class ResultReporterTests
         [OneTimeSetUp]
         public void CreateResult()
         {
-            var mockAssembly = typeof (MockAssembly).Assembly;
-            var emptySettings = new Dictionary<string, object>();
+            var mockAssembly = typeof(MockAssembly).Assembly;
+            var emptySettings = new Dictionary<string, object>
```

Comment:
Since they are no longer empty, perhaps a different name would fit better. 😄

Reply:
Whoops! 😊 I'll fix that one!